src/main/resources/database/postgres/1/patch_20.sql

```diff
@@ -158,3 +158,11 @@ BEGIN
     RETURN jsonb_pretty(_result);
 END;
 $BODY$;
+
+CREATE TABLE IF NOT EXISTS repbot_schema.message_States
```

Comment:
I think this can get confusing if we have `message_settings` which refers to `Reputation` settings and `message_states` which refers to `Message` settings.

And you have a capitalization mistake here
```suggestion
CREATE TABLE IF NOT EXISTS repbot_schema.message_states
```